AGPs/AGP-XX.md

```diff
@@ -0,0 +1,56 @@
+
+| AGP | Title | Author | Status | Track   | Created |
+|-----|-------|---------------------------|--------|---------|---------|
+| XX  | Nest budget |   louisgrx |      | Finance | 2019-07-10       |
+
+
+# AGP-XX: Nest 2019 Budget Extension
+
+## Address of the transfer recipient
+
+Nest DAO: https://mainnet.aragon.org/#/nest/0x4cbabbe13cc4b0c724e7dcecd6ae56ae1905efb2
+
+## Amount of the transfer
+
+$600k in DAI 
+
+ANT300k
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+A one-off transfer
+
+## Purpose of the transfer
+
+#### Use of funds
+The funds received will be managed by the [Nest program](https://github.com/aragon/nest) and allocated to grantees. 
+
+- $500K DAI  to cover the cost of their operations
+- $100K DAI to cover the cost of audits of successful projects
+- ANT300K to reward grantees and invite them to participate actively in Aragon
+
+#### Following up on the [AGP-17](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-17.md) budget
+In the first half of 2019, the Nest program allocated 3 grants. Considering the current pipeline of high quality proposals, it is expected by the Aragon Association that all remaining funds will have dried up by ANV#3.
```

Comment:
```suggestion
In the first half of 2019, the Nest program allocated 3 grants. Considering the current pipeline of high quality proposals, it is expected by the Aragon Association that all remaining funds will have dried up by ANV-3.
```

Reply:
The shorthand convention the community seems to have converged on is `ANV-[#]` e.g. ANV-3.